### ISSUE TITLE
Installation of dependency `coq-mathcomp-zify` on arch.

### ISSUE THREAD

**Issue by Jorispilot** [CohenCyril/lip-ssr-tuto#1]
Following the installation procedure, I came upon
```fish
❯ opam install --deps-only .
[ERROR] Package conflict!
  * Missing dependency:
    - coq-mathcomp-zify = 1.5.0
    no matching version

No solution found, exiting
```
Searching for a solution, I get
```fish
❯ opam install coq-mathcomp-zify
The following actions will be performed:
=== install 14 packages
  ∗ atd                    2.15.0         [required by atdgen, atdts]
  ∗ atdgen                 2.15.0         [required by elpi]
  ∗ atdgen-runtime         2.15.0         [required by atdgen]
  ∗ atdts                  2.15.0         [required by elpi]
  ∗ biniou                 1.2.2          [required by atdgen]
  ∗ coq-elpi               2.2.3          [required by coq-hierarchy-builder]
  ∗ coq-hierarchy-builder  1.7.0          [required by coq-mathcomp-ssreflect]
  ∗ coq-mathcomp-algebra   2.2.0          [required by coq-mathcomp-zify]
  ∗ coq-mathcomp-fingroup  2.2.0          [required by coq-mathcomp-algebra]
  ∗ coq-mathcomp-ssreflect 2.2.0          [required by coq-mathcomp-zify]
  ∗ coq-mathcomp-zify      1.5.0+2.0+8.16
  ∗ easy-format            1.3.4          [required by atd]
  ∗ elpi                   1.19.6         [required by coq-mathcomp-ssreflect]
  ∗ re                     1.12.0         [required by elpi]
```
What does `1.5.0+2.0+8.16` means? Should I install this version?

**Comment by Tragicus** [comment]
I could not confirm with Cyril yet, but `1.5.0+2.0+8.16` is the version `1.5.0` of `coq-mathcomp-zify`, so I think you can proceed.

**Comment by CohenCyril** [comment]
(closed by #2)
